### ISSUE TITLE
Incorrect source detection

### ISSUE THREAD

**Issue by MrD-RC** [FrSkyRC/ETHOS-Feedback-Community#2378]
This issue definitely effects logic switches. I don't know where else it may be effected.

If you set up a trainer. On the master transmitter, set up a logic switch to detect movement of the `Analogs` > `Aileron` source. If you move the aileron on the slave transmitter, this picked up by the `analogs`. It shouldn't be!

Analog should be the physical controls on the master transmitter. To test the slave transmitter, `Trainer` > _Channelx_ should be used for the source.

**Comment by mawzthefinn** [comment]
This is not incorrect source issues, but incorrect configuration of trainer to match requirements. 

By default, Trainer is configured to replace the sticks, not the channels, to preserve mixing and trim on the Master transmitter and not require any mixing/trim on the student radio.

This is configurable, so you can have the trainer replace channels instead of analogs, or test against the trainer channels directly. If replacing the channels, then the student radio needs full mixing/trims.

**Comment by MrD-RC** [comment]
IMHO the trainer should never replace the analogs. The analogs should always be the values output by the analog controls on the transmitter. Anything else is not logical at all.

**Comment by mawzthefinn** [comment]
Inputs is not ‘not accessible’ in ETHOS, but rather it doesn’t exist at all. There’s only Analogs and Mixers and the Mixers flow directly from Analogs. 

Once we have selectable sources for the core surface mixers this will be easier to address, but with the flow right now it HAS to function as it does, there is no step between Analogs and Mixers to address the trainer issue.

The simplest solution is to break the core surface mixers away to High Channels and treat them like the Inputs layer. But that to some extent needs Trim to be disconnectable so we can separate trims from them.

**Comment by bsongis-frsky** [comment]
The option "Ignore trainer input" is there for this purpose!

[image](https://user-images.githubusercontent.com/50834778/215525624-c9be7d39-952e-4a4b-b41d-e874c76623c8.png)

**Comment by mawzthefinn** [comment]
Completely forgot that setting exists. Yes, that solves the issue as well.

**Comment by MrD-RC** [comment]
Amazing. I feel like there needs to be a list of all these hidden commands. There is so much stuff hidden away.
